vspec2yaml.py

```diff
@@ -92,36 +79,33 @@ def write_line_break(self, data=None):
 
 
 if __name__ == "__main__":
-    #
-    # Check that we have the correct arguments
-    #
-    opts, args = getopt.getopt(sys.argv[1:], "sI:", ["no-uuid"])
-    strict = False
-    output_uuid = True
-
-    # Always search current directory for include_file
+    # The arguments we accept
+
+    parser = argparse.ArgumentParser(description='Convert vspec to yaml.')
+    parser.add_argument('-I', '--include-dir', action='append',  metavar='dir', type=str,
+                    help='Add include directory to search for included vspec files.')
+    parser.add_argument('-s', '--strict', action='store_true', help='Use strict checking: Terminate when non-core attribute is found.' )
+    parser.add_argument('--no-uuid', action='store_true', help='Exclude uuids from generated yaml.' )
+    parser.add_argument('vspec_file', metavar='<vspec_file>', help='The vehicle specification file to convert.')
+    parser.add_argument('yaml_file', metavar='<yaml file>', help='The file to output the YAML objects to.')
+
+    args = parser.parse_args()
+
+    strict=args.strict
+    generate_uuids=not args.no_uuid
```

Comment:
^

Reply:
@SebastianSchildt -will you fix what Gunnar detected and update the PR?

Reply:
Fixed